ludolph/bot.py

```diff
@@ -609,14 +610,24 @@ def _room_config(self):
             logger.error('Could not configure MUC room member list. Error was: %s (condition=%s, etype=%s)',
                          e.text, e.condition, e.etype)
 
+    def _sleekxmpp_fix_jid(self, jid):
+        """
+        SleekXMPP in version 1.3.1 returned JID object but in version 1.3.2 returned string. Backward compatible fix!
+        """
+        if isinstance(jid, (str, unicode)):
```

Comment:
This is not Python 3 compatible. You can switch the condition and test whether the jid is an instance of `JID`.
Also please move the import to the file header.